controllers/release/controller.go

```diff
@@ -81,7 +81,8 @@ func (c *Controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		adapter.EnsureReleaseIsValid,
 		adapter.EnsureFinalizerIsAdded,
 		adapter.EnsureReleaseExpirationTimeIsAdded,
-		adapter.EnsureReleaseIsProcessed,
+		adapter.EnsureTenantReleasePipelineIsProcessed,
```

Comment:
This is become a mouthful. What do you think about renaming to `Ensure{Tenant|Managed}PipelineIsProcessed`?

Reply:
makes sense. I wil do it.

Reply:
fixed